it-and-security/lib/windows/scripts/verify-windows-ca-2023.ps1

```
@@ -0,0 +1,141 @@
+
+function Write-State {
+    param([string]$Label, [string]$Value)
+    Write-Output ("{0,-32} : {1}" -f $Label, $Value)
+}
+
+Write-Output "=== Windows Secure Boot CA 2023 verification ==="
+Write-Output ""
+
+# --- Secure Boot enabled ---
+$sb = $null
+try { $sb = Confirm-SecureBootUEFI } catch {}
```

PSScriptAnalyzer (1.25.0)

[warning] Empty catch block is used. Please use Write-Error or throw statements in catch blocks.

(PSAvoidUsingEmptyCatchBlock)